src/compiler/types.ts

```diff
@@ -3691,7 +3691,7 @@ namespace ts {
 
         // Error handling
         AllowThisInObjectLiteral                = 1 << 15,
-        AllowQualifedNameInPlaceOfIdentifier    = 1 << 16,
+        AllowQualifiedNameInPlaceOfIdentifier    = 1 << 16,

```

Comment:
In #37578 I asked to deprecate misspelled variable instead of removing it, as it may break backward-compatibility (in linters, etc.)

It was previously done here, for example:
https://github.com/microsoft/TypeScript/blob/a1c8608f681488fda3f0b6ffd89195a81f80f0b0/src/compiler/types.ts#L3380-L3381

What do you think about it?

Reply:
Oh yeah, that sounds much better than deleting it. I just updated my code to deprecate it instead.